### PR TITLE
Added reviews

### PR DESCRIPTION
**Pull request by vishakahegde**
## What this pull request does

- Added reviews to the homepage
- Styled a bit

# Issues

- Slideshow needs a bit of tweaking
- It can be done later too

### CODE REVIEW

**Comment by mitraws** [comment]
Plz show us a screenshot next time =D

**Review by mitraws — APPROVED** [review]
Someone else also approve and merge plz. @jendhordejan

File: src/Components/Pages/Home/HomePage.css [hunk 1]
```diff
@@ -2,41 +2,39 @@
...
-    float: left;
-    width: 33.33%;
-    padding: 5px;
+  float: left;
+  width: 33.33%;
+  padding: 5px;
 }
 
-
 /* div {
```
Comment (jendhordejan) [inline src/Components/Pages/Home/HomePage.css:10]: please delete

File: src/Components/Pages/Home/HomePage.css [hunk 2]
```diff
@@ -2,41 +2,39 @@
...
+  font-family: "Antic Didone", "system-ui", "Segoe UI", "Roboto", "Helvetica",
+    "Arial", sans-serif;
+  opacity: 0.9;
 }
 
-
 /* Clear floats after image containers */
 
-
 /* .row::after {
```
Comment (jendhordejan) [inline src/Components/Pages/Home/HomePage.css:38]: please delete if not needed

**Review by jendhordejan — CHANGES_REQUESTED** [review]
overall the code looks good but please clean-up unncessary commented lines. if it's really needed to be left there. I suggest putting a note for the reason

File: src/Components/Pages/Home/HomePage.css [hunk 3]
```diff
@@ -53,80 +51,82 @@ h4 {
...
     border: 2px solid black;
     background-color: red; */
-    /* display: grid;
+  /* display: grid;
     grid-template-columns: repeat(3, 1fr);
     justify-content: space-around;
     align-items: center;
     padding: 20px; */
-    /* grid-template-rows: 1fr; */
+  /* grid-template-rows: 1fr; */
```
Comment (jendhordejan) [inline src/Components/Pages/Home/HomePage.css:70]: please delete if not needed

Comment (mitraws) [reply src/Components/Pages/Home/HomePage.css:70]: Will do during refactor.

**Pull request merged**